tests/extensions/Statiq.Lunr.Tests/GenerateLunrIndexFixture.cs

```diff
@@ -541,7 +550,7 @@ public async Task ConvertsSearchableField(object value, string expected)
 
                 // Then
                 TestDocument indexDocument = results.ShouldHaveSingleWithDestination(GenerateLunrIndex.DefaultScriptPath.ChangeExtension(".index.json"));
-                indexDocument.Content.ShouldContain(expected);
+                indexDocument.Content.ShouldContainWithoutWhitespace(expected); // ShouldContain is clipped to first 100 chars, but occurrence is beyond. This method is not clipped; Workaround
```

Comment:
This was a little more tricky - ShouldContain is (now?) clipped to 100 chars and the match is actually in the testdocument content - however all further text is cut-off with "..." as such the test fails. ShouldContainWithoutWhitespace is the only extension method, which is not clipping the actual. Tried to explain through comment, so it's not reversed as it's not very "intuitive"